tictactoe/tic.js

```diff
@@ -1 +1,63 @@
-console.log('Hello world!!')
\ No newline at end of file
+let model = ['','','','','','','','','']
+document.addEventListener('DOMContentLoaded', (event) => {
+  let turnNum = 0
+  let grid = document.getElementsByClassName('grid-container')
+  for (var i = 0; i < 9; i++){
+    grid[0].insertAdjacentHTML('beforeend', 
+      '<div class="grid-item" data-grid="' + i + '"><div class="xo xo-' +i +'" ></div></div>')
+  }
+  document.getElementsByClassName("grid-container")[0].addEventListener("click", 
+    (evt) => {
+      let elem = evt.target,
+          index = elem.getAttribute('data-grid'),
+          flag = 0
+      if (model[index] === '') {
+        let x = turnNum % 2 ? 'O' : 'X'
+        let xo = elem.getElementsByClassName('xo')[0]
+        model[index] = x
+        xo.innerHTML = x
+        turnNum++
+      }
+      if (model[0] === 'X' && model[4] === 'X' && model[8] === 'X' && flag === 0) {
```

Comment:
Don't repeat conditions. Take flag === 0 outside
```
if (!flag) {
   condition 1
   condition 2
   ....
}
```

Reply:
Instead of checking each condition with the positions, is it possible to use an array and make it shorter?